ledger/participant-integration-api/src/main/scala/platform/store/appendonlydao/DbDispatcher.scala

```diff
@@ -83,22 +82,24 @@ private[platform] object DbDispatcher {
   private val logger = ContextualizedLogger.get(this.getClass)
 
   def owner(
+      dataSource: DataSource,
       serverRole: ServerRole,
       jdbcUrl: String,
       connectionPoolSize: Int,
       connectionTimeout: FiniteDuration,
       metrics: Metrics,
-      connectionAsyncCommitMode: DbType.AsyncCommitMode,
   )(implicit loggingContext: LoggingContext): ResourceOwner[DbDispatcher] =
     for {
-      connectionProvider <- HikariJdbcConnectionProvider.owner(
+      hikariConnectionPool <- new HikariDataSourceOwner(
```

Comment:
Technically speaking we are getting here HikariDataSource, consider renaming the variable